src/vivarium_public_health/disease/special_disease.py

```diff
@@ -121,7 +130,9 @@ def on_time_step(self, event):
         #  if this is recoverable, anyone who gets lower exposure in the event goes back in to susceptible status.
         if self.recoverable:
             pop.loc[~sick, self.cause.name] = f'susceptible_to_{self.cause.name}'
+            pop.loc[~sick, self.susceptible_event_time_column] = event.time
```

Comment:
Event time columns track changes in state.  This is going to get everyone in a state and update their event time every time step, which is not what we want.